src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -107,31 +108,30 @@ export default class CardanoSignTransaction extends AbstractMethod {
 
         // validate incoming parameters
         validateParams(payload, [
-            { name: 'signingMode', type: 'number', obligatory: true },
-            { name: 'inputs', type: 'array', obligatory: true },
-            { name: 'outputs', type: 'array', obligatory: true, allowEmpty: true },
-            { name: 'fee', type: 'amount', obligatory: true },
-            { name: 'ttl', type: 'amount' },
+            { name: 'signingMode', type: 'number', required: true },
+            { name: 'inputs', type: 'array', required: true },
+            { name: 'outputs', type: 'array', required: true, allowEmpty: true },
+            { name: 'fee', type: 'uint', required: true },
+            { name: 'ttl', type: 'uint' },
```

Comment:
I can see that protobuf has this field as optional, but should it be set as required in this runtime check? I mean it used to be defined as required in flow.